virtool_core/models/genbank.py

```diff
@@ -0,0 +1,21 @@
+from pydantic import BaseModel, validator
+from typing import Optional
+
+valid_sequence = set('cagntCAGNT')
```

Comment:
1. Constants are all caps in Python convention. Better to use `frozenset` for this, I think.
2. Use double-quotes. This should be taken care of by Black.

```suggestion
VALID_SEQUENCE = set("cagntCAGNT")
```